2025-Code/src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -95,7 +99,7 @@ public double getTargetPosition() {
   }
 
   public double getInputVoltage() {
-    return m_motor1.getBusVoltage();
+    return m_motor1.getAppliedOutput();
```

Comment:
oh this is goofed now

Reply:
either rename to make it right or just remove it - didn't seem like we really used it.